File: Week01/Day04/solution.java
class Solution {
    public String convertToTitle(int columnNumber) {
        StringBuilder s = new StringBuilder();
        while (columnNumber > 0) {
            columnNumber--;
            int digit = columnNumber % 26;
            char c = (char) ('A' + digit);
            s.append(c);
            columnNumber /= 26;
        }
        return s.reverse().toString();
    }
}

File: Week01/Day04/solution.js
/**
 * @param {number} columnNumber
 * @return {string}
 */
var convertToTitle = function(columnNumber) {
    let s = '';
    while (columnNumber > 0) {
        columnNumber--; 
        let digit = columnNumber % 26;
        let c = String.fromCharCode(65 + digit); 
        s += c;
        columnNumber = Math.floor(columnNumber / 26);
    }
    return s.split('').reverse().join('');
};
